server/views/pages/incentives/startIncentivesMigrationPreview.njk

```diff
@@ -0,0 +1,128 @@
+{% extends "../../partials/layout.njk" %}
```

Comment:
might look at extracting nunchuck templates - all these pages are very similar. It, however given they are also very transient (e.g not need after final migration) it might just not be worth it